dev/com.ibm.ws.security.token.ltpa/src/com/ibm/ws/security/token/ltpa/LTPAKeyInfoManager.java

```diff
@@ -161,15 +161,15 @@ public synchronized final void prepareLTPAKeyInfo(WsLocationAdmin locService, St
         }
     }
 
-    public boolean isNotUseAfterDate(String filename, OffsetDateTime notUseAfterDateOdt) {
-        OffsetDateTime currentTime = OffsetDateTime.now(notUseAfterDateOdt.getOffset());
+    public boolean isValidUntilDate(String filename, OffsetDateTime validUntilDateOdt) {
```

Comment:
```suggestion
    public boolean isAfterValidUntilDate(String filename, OffsetDateTime validUntilDateOdt) {
```
It seems a bit confusing to say isValidUntilDate is true when the date has passed so it is no longer valid. All usages would need to be updated too